content/learn/08.contributions/04.arduino-creating-content-guide/arduino-creating-content-guide.md

```diff
@@ -0,0 +1,37 @@
+---
+title: 'Writing Content for Arduino'
+description: 'How to write content for the Arduino Documentation Website'
+tags: 
+  - Styleguide
+  - Guidelines
+---
+
+## Adding your own content
+
+You are welcome to add your own content to the Arduino Documentation repository. You can contribute with the following content:
+
+|Content|Description|Example|
+|-------|-----------|-------|
+|Tutorial|Learn how to do something.|[Control Built-in RGB LED over Wi-Fi with Nano RP2040 Connect](https://docs.arduino.cc/tutorials/nano-rp2040-connect/rp2040-web-server-rgb)|
+|Article|Learn about a specific topic.|[Multimeter Basics](https://docs.arduino.cc/learn/electronics/multimeter-basics)|
+|Nugget|Smaller tutorial with less information and more straight to the example.|[Analog Read Serial](https://docs.arduino.cc/built-in-examples/basics/AnalogReadSerial)|

```

Comment:
Should these be relative links?

Reply:
Yes they should ⭐ I haven't got the hang of them myself though, could you help or maybe @karlsoderby?

Reply:
```suggestion
|Tutorial|Learn how to do something.|[Control Built-in RGB LED over Wi-Fi with Nano RP2040 Connect](/tutorials/nano-rp2040-connect/rp2040-web-server-rgb)|
|Article|Learn about a specific topic.|[Multimeter Basics](/learn/electronics/multimeter-basics)|
|Nugget|Smaller tutorial with less information and more straight to the example.|[Analog Read Serial](/built-in-examples/basics/AnalogReadSerial)|
```
OK, here they are converted to relative.